resources/geometry.rb

```diff
@@ -19,6 +19,69 @@ def self.make_polygon(*pts)
     return p
   end
 
+  def self.get_thermal_zones(building:)
+    thermal_zones = []
+    building.elements.each("BuildingDetails/Enclosure/Walls/Wall") do |wall|
+      wall_values = HPXML.get_wall_values(wall: wall)
+      thermal_zones << wall_values[:interior_adjacent_to]
```

Comment:
The code needs to check `exterior_adjacent_to` values for all surfaces as well. There is no convention in HPXML, so a wall between garage and living space can either be interior=living space and exterior=garage OR interior=garage and exterior=living space.

Reply:
Good point. Although, I'll need to exclude "outside" and "garage" (I think).

Reply:
Garage should not be excluded, but "ground" should. Maybe that's what you meant?

Reply:
Yes, "ground" instead of "garage" is what I meant.